server/schemas/item.graphql

```diff
@@ -118,19 +118,21 @@ input ItemQuery {
 }
 
 extend type Query {
-  items(schemaId: ID!, sort: ItemSort, pagination: Pagination): ItemConnection!
+  items(modelId: ID!, pagination: Pagination): ItemConnection!
+  itemsByIds(ids: [ID!]!): ItemConnection!
   versionsByItem(itemId: ID!): [VersionedItem!]!
   searchItem(
     query: ItemQuery!
     sort: ItemSort
     pagination: Pagination
   ): ItemConnection!
+  checkIfItemIsReferenced(itemId: ID!): Boolean
```

Comment:
Please give me a use case. This query seems to have a very limited use case. I think it could be made into a more useful query.

Reply:
I am using this to check if an item is reference or not by id. 
This is useful in the frontend to give the user a warning in case of dereferencing a referenced item.
Please let me know if you have a better way to do so.